api/transcript.js

```diff
@@ -23,8 +23,6 @@ const loadTranscript = () => {
   let __dirname = decodeURIComponent(path.dirname(moduleURL.pathname));
   __dirname = os.platform() == "win32" ? __dirname.slice(1) : __dirname
   
-  console.log("Dirname = ", __dirname);
```

Comment:
@sampoder Just curious - why was this change necessary, and why was it done within the context of this PR, and not given as feedback to author?

Reply:
I want to avoid having too many console.log() statements in the codebase

Reply:
@sampoder understood - however, given that this is actually part of the PR and thus presumably desired by the author, it should probably not just be deleted without conversation.  If there's reason to believe this would be overly chatty, for instance being in some sort of tight loop, it's even more important for that to be made clear and discussed.

TLDR - I would recommend not adding "finalizing edits" on top of others' PR especially without giving them opportunity to discuss the changes.